cmd/gctcli/futures.go

```diff
@@ -298,6 +217,268 @@ var futuresCommands = &cli.Command{
 				},
 			},
 		},
+		{
+			Name:      "getcollateralmode",
+			Aliases:   []string{"gcm"},
+			Usage:     "gets the collateral mode for an exchange asset",
+			ArgsUsage: "<exchange> <asset>",
+			Action:    getCollateralMode,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:    "exchange",
+					Aliases: []string{"e"},
+					Usage:   "the exchange to retrieve futures positions from",
+				},
+				&cli.StringFlag{
+					Name:    "asset",
+					Aliases: []string{"a"},
+					Usage:   "the asset type of the currency pair, must be a futures type",
+				},
+			},
+		},
+		{
+			Name:      "setcollateralmode",
```

Comment:
utACK binance - keys

Reply:
You can test binance by using testnet keys with testnet urls in your config

Reply:
Will do that next round 👍